lang/node/Makefile

```diff
@@ -129,6 +129,7 @@ HOST_CONFIGURE_ARGS:= \
 	--dest-os=linux \
 	--without-snapshot \
 	--shared-zlib \
+	--systemtap-includes=$(HOST_BUILD_DIR)/deps/openssl/openssl/include \
```

Comment:
The configure option should really be `--shared-openssl-includes`?

Reply:
In node.js' host build we use bundle openssl instead of shared openssl.
Specifying "--shared-openssl-includes” option is ineffective useing bundle openssl.

Host build of node.js now fails after the version of libressl updated on current openwrt.

In order to avoid it, I used a dirty method.

Reply:
@nxhack , please document the intent either in the commit message or properly in the Makefile.  Thank you.

Reply:
As you point out, it is a confusing, bad workaround.

* When building bundles openssl of node.js, it fails by referring to the header of libressl.

* "configure script" of node.js generates config.gyp.
```
                       'include_dirs': [ '/home/ubuntu/Trunk/openwrt/staging_dir/host/include'],
                       'libraries': [ '-L/home/ubuntu/Trunk/openwrt/staging_dir/host/lib',
                                      '-lz']},
```

I want to change include_dirs.
I will think again whether there is a good hack.

Reply:
OK I found, pkg-config was set to build shared libz. As a workaround, I will change libz to use bundle version as well.